Makefile

```diff
@@ -82,6 +83,10 @@ lint-black:
 
 lint: lint-black lint-flake8
 
+lint-ansible:
```

Comment:
WIll those be applied to all the playbooks ?

Reply:
Yes. It loads the playbooks and in turn a 	Ansible-specific (case-agnostic): ll related roles will also be loaded and checked as well.

I wanted to add an actual linter here (ansible-linter probably), but we'll need to properly configure it first.